tests/keithley_2700.py

```diff
@@ -0,0 +1,188 @@
+import unittest
+from unittest import skipIf
+from utils.channel_access import ChannelAccess
+from utils.testing import get_running_lewis_and_ioc
+from utils.ioc_launcher import IOCRegister
+
+
+
+class Keithley_2700Tests(unittest.TestCase):
+    """
+    Tests for the Keithley2700.
+    """
+
+    def setUp(self):
+        self._lewis, self._ioc = get_running_lewis_and_ioc("keithley_2700")
+        self.ca = ChannelAccess(default_timeout=30)
+        self.ca.wait_for("KHLY2700_01:IDN")
+
+    @skipIf(IOCRegister.uses_rec_sim, "In rec sim this test fails")
```

Comment:
Using "In rec sim this test fails" is a bad habit we've gotten into. Please could we break it. Not running the test in rec sim tells me that it fails in rec sim. I'd like the commenting to tell me what it is about the test that doesn't work.